homeassistant/components/sonos/speaker.py

```diff
@@ -63,6 +67,27 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def update_alarms_for_speaker(soco: SoCo, alarms: dict[str, Alarm] = {}) -> set[str]:
```

Comment:
```suggestion
def update_alarms_for_speaker(soco: SoCo, alarms: dict[str, Alarm]) -> set[str]:
```
Default value is not needed, right?

Reply:
I think that's outdated.

Reply:
It is. I was in the middle of a review while you pushed 😄